Tests/EmbraceCoreTests/TestSupport/TestDoubles/Dummy.swift

```diff
@@ -0,0 +1,67 @@
+//
```

Comment:
Can we change this filename to match `DummyURLSessionInitWithDelegateSwizzler`

Reply:
Oh yes! Dunno why It was saved as dummy 🤔  so weird!
Gonna change it 👍